hooks/pre-push

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Check for any changes
+if [[ -n $(git status --porcelain) ]]; then
+  echo "Uncommitted changes detected! Cannot test for formatting."
+  echo "Please commit all local changes before pushing."
+  exit 1
+fi
+
+if [[ $(whoami) == 'vagrant' ]]; then
```

Comment:
Nit: Can you add an environment variable like `|| MS_RUN_NATIVE` or some such so allow me to run this outside virtual box?